content/learn/tested.adoc

```diff
@@ -0,0 +1,82 @@
+
+[id="must-tested-tier"]
+=== Must
+
+A {tested} pattern must continue to meet the following criteria to remain in the {tested} tier:
+
+* A {tested} pattern must conform to the common technical link:/requirements/implementation/[implementation requirements]
+* A {tested} pattern must be meaningful without specialized hardware, including flavors of architectures not explicitly supported.
++
+Qualification is a {solution-name-upstream} Technical Oversight Committee (TOC) decision with input from the pattern owner.
+* A {tested} pattern must have their implementation reviewed by the patterns team to ensure that it is sufficiently flexible to function across a variety of platforms, customer environments, and any relevant verticals.
+* A {tested} pattern must include a standardized architecture drawing, created with (or at least conforming to) the standard {solution-name-upstream} tooling
+* A {tested} pattern must include a written guide for others to follow when demonstrating the pattern
+* A {tested} pattern must include a test plan covering all features or attributes being highlighted by the demonstration guide. Negative flow tests (such as resiliency or data retention in the presence of network outages) are also limited to scenarios covered by the demonstration guide.
++
+The test plan must define how to validate if the pattern has been successfully deployed and is functionally operational.
+Example: https://docs.google.com/document/d/12KQhdzjVIsxRURTnWAckiEMB3_96oWBjtlTXi1q73cg/view[Validating an Industrial Edge Deployment]
++
+//TODO: Convert above link to adoc
+
+* A {tested} pattern must nominate at least one currently supported {rh-ocp} release to test against.
+* A {tested} pattern must ensure the test plan passes at least once per quarter.
+* A {tested} pattern must create a publicly available JSON file (eg. in an AWS bucket) that records the result of the latest test for each combination of pattern, platform, and {rh-ocp} version. See link:/learn/test-artefacts/[testing artefacts]
+
+[NOTE]
+====
+The {tested} pattern *do not* imply an obligation of support for partner or community operators by Red Hat or the pattern owner.
+====
+
+[id="should-tested-tier"]
+=== Should
+
+* A {tested} pattern should be broadly applicable.
+* A {tested} pattern should focus on functionality not performance.
+
+[id="can-tested-tier"]
+=== Can
+
+Teams creating {tested} pattern can provide their own service level agreement (SLA).
+
+A technical document for Quality Engineering (QE) team that defines how to validate if the pattern has been successfully deployed and is functionally operational.
+For example, see https://docs.google.com/document/d/12KQhdzjVIsxRURTnWAckiEMB3_96oWBjtlTXi1q73cg/view[Validating an Industrial Edge Deployment].
+
+* A {tested} pattern meeting additional criteria can be nominated for promotion to the link:/learn/maintained/[Maintained tier].
```

Comment:
Is this bullet point here required since the rest of this section is unbulleted?